### DISCUSSION TITLE
Can't replace logo in Organisation

### DISCUSSION THREAD

**Discussion by stellarfinn** [Budibase/budibase#4686] [Help] [answered]
On Budibase Cloud. 

I've tried removing and uploading a new logo that is square instead of horizontal and the preview doesn't show up. I hit save. Refresh and the old horizontal logo shows up.

Thoughts?

**Comment by stepcellwolf** [comment]
Hi,

I'm having the same issue? Any update on the fix?

Thanks in advance!

**Comment by melohagan** [comment, marked as answer]
Hey @stepcellwolf 

Removing the Budibase logo in cloud is part of the pro plan: https://budibase.com/pricing/
